auto_submit/lib/validations/ci_successful.dart

```diff
@@ -128,6 +130,13 @@ class CiSuccessful extends Validation {
     for (ContextNode status in statuses) {
       // How can name be null but presumed to not be null below when added to failure?
       final String? name = status.context;
+
+      // If the account author is a roller account do not block merge on flutter-gold check.
+      if (config.rollerAccounts.contains(author.login!) && slug == Config.engineSlug && name == 'flutter-gold') {
```

Comment:
@zanderso Do we want to apply the rule to only skia/dart rollers, or to all rollers including the `dependabot` for bumping dependencies?

Reply:
If possible, yeah, dependabot should also not be blocked on skia gold diffs.